package/network/services/hostapd/files/hostapd.sh

```diff
@@ -752,7 +752,8 @@ hostapd_set_bss_options() {
 		}
 	}
 
-	append bss_conf "ssid=$ssid" "$N"
+	[ "x${ssid}" = 'x' ] || append bss_conf "ssid=${ssid}" "$N"
```

Comment:
Why do you use it this way rather than `[ -n "$ssid"] &&`?

Reply:
@aparcar 
My bad -- used the most `sh`-portable out of a habit. Shall I change that?